css/style.css

```diff
@@ -0,0 +1,61 @@
+body {
+  font-family: sans-serif;
+  border-top: 5px solid black;
+  border-bottom: 5px solid black;
+  padding-top: 50px;
+  padding-bottom: 50px;
+}
+
+nav {
+  text-align: center;
+}
+
+nav > a {
+  color: black;
+}
+
+p > a {
+  color: black;
+  text-transform: uppercase;
+  font-size: 11px;
+}
+
+h1 {
+  text-transform: uppercase;
+  text-align: center;
+  font-size: 25px;
+}
+
+h2 {
+  font-size: 20px;
+}
+
+div {
+  margin: 0 auto;
+  width: 600px;
+}
+
+img.bio {
+  margin-top: 5px;
+  width: 150px;
+  float: right;
+  padding: 5px;
+}
+
+img.interests{
```

Comment:
@andrewhinderaker  in the future when adding a class to an element just use the class by itself as the selector. Unless you need to add more specificity to the selector.